src/components/e-fullwidth.vue

```diff
@@ -71,29 +79,31 @@
       }
     },
     mounted() {
-      if (!this.resizeObserver) { // IE11 fallback.
+      if (!this.resizeObserver) {
         this.updateScrollbarWidth();
       }
     },
     // beforeUpdate() {},
     // updated() {},
     // activated() {},
     // deactivated() {},
-    beforeDestroy() {
+    beforeUnmount() {
       if (this.resizeObserver) {
         this.resizeObserver.unobserve(window.document.body);
-      } else { // IE11 fallback.
+      } else {
```

Comment:
This scenario can be removed, since all modern browsers know the resizeObserver.